Sources/Testing/Testing.docc/Traits.md

```diff
@@ -10,14 +10,14 @@ See https://swift.org/LICENSE.txt for license information
 See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 
-Add traits to tests to annotate them or customize their behavior.
+Annotate tests and suites, and customize their behavior.
```

Comment:
I don't think we're entirely consistent here, but I've usually been writing "test functions and [test] suites". We should decide what phrasing we prefer and apply it consistently.

Reply:
I think "test functions and suites" is fine, we don't need the repetition of "test functions and test suites", I'll use that.